srcpkgs/gnome-flashback/template

```diff
@@ -0,0 +1,31 @@
+# Template file for 'gnome-flashback'
+pkgname=gnome-flashback
+version=3.30.0
+revision=1
+only_for_archs="i686 x86_64"
+build_style=gnu-configure
+configure_args="--prefix=/usr --sysconfdir=/etc --localstatedir=/var --libexecdir=/usr/lib/${pkgname} --disable-schemas-compile --enable-compile-warnings=minimum --enable-gtk-doc"
+hostmakedepends="automake autogen autoconf-archive gettext-devel intltool libtool git gobject-introspection gtk-doc intltool libxslt pkg-config
+python3 sassc"
```

Comment:
`configure_args`, `hostmakedepends`, and `makedepends` are not properly formatted.

Reply:
How should I format them properly?

Reply:
Look at the other templates...